lesson21/index.js

```diff
@@ -110,5 +112,60 @@ const createTd = (usersData, keys) => {
   return fragment;
 }
 
+const createSortBtn = () => {
+  const sortBtn = createElementWithClassName("button", "sort-btn");
+  sortBtn.dataset.sortStatus = "standard";
+  sortBtn.id = "js-sort-btn";
+  return sortBtn;
+}
+
+const setClickInSortBtn = () => {
+  const standardRows = [...document.querySelector("tbody").querySelectorAll("tr")];
+  const sortBtn = document.getElementById("js-sort-btn");
+
+  sortBtn.addEventListener("click", (e) => {
+    let currentStatus = e.target.dataset.sortStatus;
+    currentStatus = switchSortStatus(currentStatus);
+    e.target.dataset.sortStatus = currentStatus;
+
+    const sortedRows = getSortedRows(currentStatus, standardRows);
+
+    const tbody = document.querySelector("tbody");
+    sortedRows.forEach((row) => {
+      tbody.appendChild(row);
+    })
+  });
+}
+
+const switchSortStatus = (status) => {
+  if (status === "standard") {
+    return "asc";
+  } else if (status === "desc") {
+    return "standard";
```

Comment:
It gave me a chance to investigate the difference between default and standard.
I thought default was more appropriate here since I wanted to display the data in the order of response from the server.
1ce5df2